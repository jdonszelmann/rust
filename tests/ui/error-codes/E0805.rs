#![feature(eii)]

#[eii(foo)]
fn x();

#[foo]
fn y(a: u64) -> u64 {
//~^ ERROR E0805
    a
}

fn main() {}
