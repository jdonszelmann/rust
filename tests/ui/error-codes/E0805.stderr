error: free function without a body
  --> $DIR/E0805.rs:4:1
   |
LL | fn x();
   | ^^^^^^-
   |       |
   |       help: provide a definition for the function: `{ <body> }`

error: cannot find attribute `foo` in this scope
  --> $DIR/E0805.rs:6:3
   |
LL | #[foo]
   |   ^^^

error: cannot find attribute `eii` in this scope
  --> $DIR/E0805.rs:3:3
   |
LL | #[eii(foo)]
   |   ^^^

error[E0635]: unknown feature `eii`
  --> $DIR/E0805.rs:1:12
   |
LL | #![feature(eii)]
   |            ^^^

error: aborting due to 4 previous errors

For more information about this error, try `rustc --explain E0635`.
